mnemonic-computing-services/mnemonic-utilities-service/build.gradle

```diff
@@ -26,7 +26,6 @@ description = 'mnemonic-utilities-service'
 dependencies {
   implementation project(':mnemonic-core')
   implementation project(':mnemonic-collections')
-  implementation 'org.flowcomputing.commons:commons-primitives'
```

Comment:
By removing this dependency, the gradle may not build correctly as new dependency of mnemonic-common is not present. Please double check